scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -134,7 +134,7 @@ package com.twitter.scalding {
     val boxedSemigroup = Externalizer(commutativeSemigroup)
 
     val DEFAULT_CACHE_SIZE = 100000
-    val SIZE_CONFIG_KEY = AggregateBy.AGGREGATE_BY_THRESHOLD
+    val SIZE_CONFIG_KEY = AggregateByProps.AGGREGATE_BY_CAPACITY
```

Comment:
Trick is, AggregateBy.AGGREGATE_BY_THRESHOLD has disappeared from the Cascading API. We can of course read it and carry that value forward if AggregateByProps.AGGREGATE_BY_CAPACITY has been left unset by the user; is that what you'd like me to do?

Do you want me to bring that constant back (within scalding only), take AggregateByProps.AGGREGATE_BY_CAPACITY if defined or else take AggregateBy.AGGREGATE_BY_THRESHOLD (logging a warning) ?

@johnynek just to make sure I understand, when you write " (instead explain what it means to the user, not necessarily in cascading terms)", do you mean logging a warning, or leaving a comment in the code?